Frontend/ui-customer/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -432,7 +432,8 @@ config =
   }
   , enableBookAny : true
   , acPopupConfig : {
-      enable : false,
```

Comment:
don't remove existing keys from config, this may lead to backward compatibility issues, instead add a new one and deprecate it later

Reply:
here it will be fine, as it's being added for the first time